Makefile

```
@@ -8,7 +8,7 @@ IMAGE_BUILD_CMD ?= docker build
 IMAGE_BUILD_EXTRA_OPTS ?=
 IMAGE_PUSH_CMD ?= docker push
 CONTAINER_RUN_CMD ?= docker run
-BUILDER_IMAGE ?= golang:1.23-bookworm
+BUILDER_IMAGE ?= golang:1.24rc3-bookworm
```

@fmuyassarov  I think the docker images are now available to non-rc image

```suggestion
BUILDER_IMAGE ?= golang:1.24-bookworm
```

---

Cool, updated now.